server/app/services/WellKnownPaths.java

```diff
@@ -9,4 +9,5 @@ public class WellKnownPaths {
   public static Path APPLICANT_MIDDLE_NAME = Path.create("applicant.name.middle_name");
   public static Path APPLICANT_LAST_NAME = Path.create("applicant.name.last_name");
   public static Path APPLICANT_NAME = Path.create("applicant.name");
+  public static Path APPLICANT_DOB = Path.create("applicant.date_of_birth");
```

Comment:
question (blocking): confirm that in Seattle's system the DOB question has this path